### DISCUSSION TITLE
Which combination is the best one?

### DISCUSSION THREAD

**Discussion by Hubit1024** [Chia-Network/chia-blockchain#1741] [General]
I have 9T SSD, and my computer is good enough.
Option 1: plot size, I chose K32 plot (101GB--332GB); plot number, I choose 8 plot, total 808-2656GB
Option 2: Plot size, Ichoose K35 plot(884.1GB--2355GB); plot number, I choose 1 plot, total 884.1-2355GB
I am confused which one is better? Thx

**Comment by hoffmang9** [comment]
You should almost always plot k=32.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
